### PR TITLE
Add seed_run_results model

### PR DESCRIPTION
**Pull request by kokorin**

### CODE REVIEW

File: models/edr/run_results/seed_run_results.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    run_results.execute_started_at,
+    run_results.execute_completed_at,
+    run_results.compile_started_at,
+    run_results.compile_completed_at,
+    run_results.compiled_code,
+    run_results.adapter_response,
+    run_results.thread_id,
+    seeds.database_name,
+    seeds.schema_name,
+    coalesce(run_results.materialization, seeds.materialization) as materialization,
```
Comment (ofek1weiss) [inline models/edr/run_results/seed_run_results.sql:36]: looks like the `dbt_seeds` model does not have a `materialization` column, you can just do 
```suggestion
    run_results.materialization,
```

Comment (kokorin) [reply models/edr/run_results/seed_run_results.sql:36]: Thank you, all test passed locally, I will check why

Comment (kokorin) [reply models/edr/run_results/seed_run_results.sql]: Added tests to validate elementary view models including `seed_run_results`.

**Review by ofek1weiss — APPROVED** [review]
Very nice, approved 👍 🚀

**Pull request merged**